cabal.project

```diff
@@ -47,15 +53,15 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-db-sync
-  tag: 1185967393d5052f3c5af540e1e1f058599019cf
-  --sha256: 0irjm0ji7kdmaa25d9248kcy78a19akz82zvrrxh78r496i0cch2
+  tag: fe1bcc55c857cbebbf331d81383fd6608f723fa1
```

Comment:
Can we make sure that the same revisions are also updated in the stack's snapshot?

Reply:
WE say in release notes that it is compatible with cardano-db-sync 3.0.0 that has the later commit https://github.com/input-output-hk/cardano-db-sync/commit/beca3f8e1868bcc12da2ca4bbf0f97708fefc6c3.